processor/onboarding/onboarding.go

```diff
@@ -0,0 +1,33 @@
+package onboarding
```

Comment:
I would suggest not to put this under `processors` namespace but only `onboarding`. Reason is that from my point of view it doesn't have the typical properties of a processor like and endpoint handler or any special fields.